e2e/e2e_metric_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build bcc
```

Comment:
keep this to avoid getting tested on mac

Reply:
I will fix this one later.
the major issue is about the test scope.
as matrix * pod * namespace?

Reply:
yes, we need to validate for each pod, the kepler metrics are created and hopefully non-zero.